README.md

```diff
@@ -31,9 +31,23 @@ Assigns all local variables into the scratch pad.
 Originally implemented and named by Antoine Levitt in [Exfiltrator.jl](https://github.com/antoine-levitt/Exfiltrator.jl).
```

Comment:
I appreciate the credit (especially for the name, which I was particularly happy about) but that's really not necessary and doesn't add information to the user, so I'd just remove it